govuk_frontend_jinja/templates/macros/i18n.html

```diff
@@ -0,0 +1,15 @@
+{#
+ # Renders the data attributes for the different plural forms
+ # of the given translation key.
+ #
+ # Helps reduce the boilerplate in component templates as they're quite verbose
+ #
+ # @private
+ # @param {string} translationKey - The kebab-cased name of the translation key
+ # @param {object} pluralForms
+ #   An object associating translation messages to the plural form they correspond to
+ #   http://unicode.org/reports/tr35/tr35-numbers.html#Language_Plural_Rules
+ #}
+{% macro govukPluralisedI18nAttributes(translationKey, pluralForms) %}
+    {% for pluralType, message in pluralForms %} data-i18n.{{translationKey}}.{{pluralType}}="{{message | escape}}"{% endfor %}
```

Comment:
Ooops, forgot this suggestion to fix the tuple expansion test failure. As a note this makes pluralForms required, which I'm pretty sure is fine but just wanted to raise with you in case someone might want to pass in a falsey value (but right now the character-count component always has a falsey guard around this macro's invocation)

```suggestion
    {% for pluralType, message in pluralForms.items() %} data-i18n.{{translationKey}}.{{pluralType}}="{{message | escape}}"{% endfor %}
```